lib/aio/job.py

```diff
@@ -150,6 +152,7 @@ async def run_container(job: Job, subject: Subject, ctx: JobContext, log: LogStr
                     raise Failure(f'Container exited with code {returncode}')
 
             finally:
+                logger.debug('force removing container')
```

Comment:
Ah, is it that removal which fails? I suggest to also log if `cidfile` exists at that point.